src/popup/snooze-content.js

```diff
@@ -53,9 +54,11 @@ function undeleteSnoozedTab(item) {
 }
 
 function openSnoozedTab(item) {
+  log('openSnoozedTab', item);
```

Comment:
Is this log intentional or remaining from debugging?

Reply:
Whoops! Removed!